paperclip
=========

lists of papers on different topics

## Reviews: X chromosome inactivation
* [X inactivation and the complexities of silencing a sex chromosome (Chow and Heard, 2009)](http://www.sciencedirect.com/science/article/pii/S0955067409000982)  includes an examination of the sequence of epigenetic events accompanying the silencing of the X chromosome during differentiation. 
* [X-inactivation, imprinting, and long noncoding RNAs in health and disease (Lee and Bartolomei, 2013)](http://www.cell.com/abstract/S0092-8674%2813%2900205-5)

## Review: Sperm chromatin
* [Unique Chromatin Remodeling and Transcriptional Regulation in Spermatogenesis (Sassone-Corsi 2002)](http://science.sciencemag.org/content/296/5576/2176.full)

## Active demethylation with TET, AID, TDG and BER proteins
### Review: [DNA Demethylation Pathways: Recent Insights (Li 2013)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4222334/)
* Glycosylase-dependent DNA demethylation was first proposed in animals.
  - [Mechanisms of DNA Demethylation in Chicken Embryos PURIFICATION AND PROPERTIES OF A 5-METHYLCYTOSINE-DNA GLYCOSYLASE (Jost 1995)](http://www.jbc.org/content/270/17/9734.short)
  - [5-methylcytosine-DNA glycosylase activity is present in a cloned G/T mismatch DNA glycosylase associated with the chicken embryo DNA demethylation complex (Zhu 2000)](http://www.pnas.org/content/97/10/5135.long)
    - The 5-methylcytosine glycosylase activity was initially detected in chicken embryo extracts, which contains TDG
    - the glycosylase activity of TDG is much lower against 5-methylcytosine than against mismatched thymine
  - [5-Methylcytosine DNA glycosylase participates in the genome-wide loss of DNA methylation occurring during mouse myoblast differentiation (Jost 2001)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC60186/)
* [LCX, Leukemia-associated Protein with a CXXC Domain, Is Fused to MLL in Acute Myeloid Leukemia with Trilineage Dysplasia Having t(10;11)(q22;q23) (Ono 2002)](http://cancerres.aacrjournals.org/content/62/14/4075.short)
  -  **TET was identified** the TET1 gene, located on chromosome 10, can translocate with the H3K4 histone methyltransferase MLL gene on chromosome 11
* [DNA demethylation in zebrafish involves the coupling of a deaminase, a glycosylase, and gadd45. (Rai 2008)](http://www.sciencedirect.com/science/article/pii/S0092867408015171)
  - Using zebrafish embryos we provide evidence for 5-methylcytosine (5-meC) removal in vivo via the coupling of a 5-meC deaminase (AID, which converts 5-meC to thymine) and a G:T mismatch-specific thymine glycosylase (Mbd4).
  - Our results provide **evidence for a coupled mechanism of 5-meC demethylation, whereby AID deaminates 5-meC, followed by thymine base excision by Mbd4, promoted by Gadd45**.
* [Reprogramming towards pluripotency requires AID-dependent DNA demethylation (Bhutani 2009)](https://www.nature.com/nature/journal/v463/n7284/full/nature08752.html)
  - mammalian **AID is required for active DNA demethylation** and initiation of nuclear reprogramming towards pluripotency in human somatic cells
* [The Nuclear DNA Base 5-Hydroxymethylcytosine Is Present in Purkinje Neurons and the Brain (Kriaucionis & Heintz 2009)](http://science.sciencemag.org/content/324/5929/929.long)
  - **detected the presence of 5-hydroxymethyl-2′-deoxycytidine (hmdC)** in neuronal cells.
  - hmdC constitutes 0.6% of total nucleotides in Purkinje cells, 0.2% in granule cells, and is not present in cancer cell lines.
* [Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. (Tahiliani 2009)](http://science.sciencemag.org/content/324/5929/930)
  - In a computational search for enzymes that could modify 5-methylcytosine (5mC), we identified TET proteins as mammalian homologs of the trypanosome proteins JBP1 and JBP2, which have been proposed to oxidize the 5-methyl group of thymine.
  - **TET1** is a 2-oxoglutarate (2OG)- and Fe(II)-dependent enzyme that **catalyzes conversion of 5mC to 5-hydroxymethylcytosine (hmC)** in cultured cells and in vitro
* [Hydroxylation of 5-Methylcytosine by TET1 Promotes Active DNA Demethylation in the Adult Brain (Guo 2011)](http://www.cell.com/abstract/S0092-8674(11)00299-6)
  - 5mC hydroxylation promotes active DNA demethylation irrespective of CpG context
  - **Deamination and base excision repair are involved in 5hmC demethylation**
  - **5hmC demethylation is highly processive, transcription dependent, and strand biased**
  - Tet1 and Apobec1 regulate activity-induced DNA demethylation in the mouse brain
* [ Tet proteins can convert 5-methylcytosine to 5-formylcytosine and 5-carboxylcytosine. (Ito 2011)](http://science.sciencemag.org/content/333/6047/1300.long)
  - the **Tet proteins can generate 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC) from 5mC** in an enzymatic activity–dependent manner.
* [Tet-Mediated Formation of 5-Carboxylcytosine and Its Excision by TDG in Mammalian DNA (He 2011)](http://science.sciencemag.org/content/333/6047/1303)
  - we demonstrate that 5mC and 5hmC in DNA are oxidized to 5-carboxylcytosine (5caC) by Tet dioxygenases in vitro and in cultured cells.
  - **5caC is specifically recognized and excised by thymine-DNA glycosylase (TDG)**.
  - Depletion of TDG in mouse embyronic stem cells leads to accumulation of 5caC to a readily detectable level.
  - These data suggest that oxidation of 5mC by Tet proteins followed by TDG-mediated base excision of 5caC constitutes a pathway for active DNA demethylation.
* [ Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. (Cortellino 2011)](http://www.sciencedirect.com/science/article/pii/S0092867411006623)
  - Inactivation of TDG leads to embryonic lethality and altered DNA methylation patterns
  - TDG keeps CpG islands unmethylated and actively demethylates promoters and enhancers
  - TDG interacts with AID and GADD45a and regulates the levels of AID
  - TDG removes 5-hydromethyluracil originated by deamination of 5-hydroxymethylcytosine
* TAB-seq
  - [Tet-assisted bisulfite sequencing of 5-hydroxymethylcytosine. (Yu 2012)](https://www.nature.com/nprot/journal/v7/n12/abs/nprot.2012.137.html)
  - [Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome. (Yu 2012)](http://www.sciencedirect.com/science/article/pii/S009286741200534X)
* [Mechanism and stem-cell activity of 5-carboxycytosine decarboxylation determined by isotope tracing (Schiesser 2012)](http://onlinelibrary.wiley.com/doi/10.1002/anie.201202583/full)
  - this mechanism allows exchange of mC by dC without formation of intermediate strand breaks
* [Genome-wide analysis reveals TET- and TDG-dependent 5-methylcytosine oxidation dynamics. (Shen 2013)](http://www.sciencedirect.com/science/article/pii/S0092867413004017)
  - Modification-specific antibodies reveals TET/TDG-mediated 5mC oxidation dynamics
  - A resource of genome-wide distribution for all oxidized forms of 5mC in mouse ESCs
  - Tdg depletion induces 5fC and 5caC accumulation at bivalent and silent promoters
  - TDG activity is preferentially recruited to distal cis-regulatory elements
* [Genome-wide Profiling of 5-Formylcytosine Reveals Its Roles in Epigenetic Priming (Song 2013)](http://www.sciencedirect.com/science/article/pii/S0092867413004005)
  - Two methods detect 5fC, an oxidized form of 5-methylcytosine, in the genome of mESCs
  - Genome-wide profiling reveals TDG-dependent regulation of 5fC, not 5hmC, in mESCs
  - 5fC is preferentially enriched at poised enhancers and correlates with p300 binding
  - Detection methods rely on selective chemical labeling and base resolution

 
## DNA methylation and DNA sequence
* [Protection of CpG islands from DNA methylation is DNA-encoded and evolutionarily conserved (Long 2016)](https://academic.oup.com/nar/article/44/14/6693/2468145/Protection-of-CpG-islands-from-DNA-methylation-is#87018343)
  - transchromosomic mouse animal model 
  - CpG-rich promoter-associated CGIs are almost invariantly hypomethylated. 
  - distal elements are prone to alternative DNA methylation states depending on the host species; this relies on both DNA sequence and transcription factor binding.
  - these observations hold true when mouse chromosomal fragments are transposed into zebrafish, demonstrating for the first time that these general mechanisms are functionally conserved across divergent vertebrate species
  
## DNA methylation and its regulatory association
* [The accessible chromatin landscape of the human genome (Thurman2012)](http://www.nature.com/nature/journal/v489/n7414/full/nature11232.html) We focused on 243,037 CpGs falling within DHSs in 19 cell types for which both data types were available from the same sample. 
  - We observed **two broad classes** of sites: (1) those with a strong inverse correlation across cell types between DNA methylation and chromatin accessibility, and (2) those with variable chromatin accessibility but constitutive hypomethylation. 
  - methylation patterning paralleling cell-selective chromatin accessibility results from passive deposition after the vacation of transcription factors from regulatory DNA

## Epigenetics and Evolution

#### General
* [Molecular basis of base substitution hotspots in Escherichia coli (Coulondre, 1978)](http://europepmc.org/abstract/MED/355893): spontaneous deamination of 5-methylcytosine to thymine 
* [The rate of hydrolytic deamination of 5-methylcytosine in double-stranded DNA (Shen, 1994)](https://academic.oup.com/nar/article/22/6/972/1032988/The-rate-of-hydrolytic-deamination-of-5) The rate constants for spontaneous hydrolytic deamination of 5-methylcytosine and cytosine in double-stranded DNA at 37°C were 5.8e−13/s and 2.6e−13/s, respectively.
* [Evolutionary changes in CpG and methylation levels in the genome of vertebrates (Jabbari, 1997)](http://www.sciencedirect.com/science/article/pii/S0378111997004757) The 5mC and CpG observed/expected values show no overlap between the two groups of vertebrates and suggest the existence of two equilibria. (higher in fishes and amphibians than in mammals and birds -- body temperature; a side point: fishes and amphibians have higher content of repetitive DNA)
* [DNA methylation and body temperature in fishes (Varriale 2006)](http://www.sciencedirect.com/science/article/pii/S037811190600518X) an inverse relationship between DNA methylation and body temperature. (lower temperature & higher %5mC, but didn't examine level of CpG decay)
* [The evolution of invertebrate gene body methylation (Sarda, 2012)](http://mbe.oxfordjournals.org/content/29/8/1907.short) analyzed methylation levels of orthologs from four distantly related invertebrate species. 
* [Gene body methylation is conserved between plant orthologs and is of evolutionary consequence (Takuno and Gaut, 2013)](http://www.pnas.org/content/110/5/1797.short)

#### Mammals
* [The human colon cancer methylome shows similar hypo-and hypermethylation at conserved tissue-specific CpG island shores (Irizarry et al. 2009)](http://www.nature.com/ng/journal/v41/n2/abs/ng.298.html) CpG island shore methylation was strongly related to gene expression, and it was highly conserved in mouse, discriminating tissue types regardless of species of origin. **CpG island shores**: in sequences up to 2 kb distant, show differential methylation in cancer, or tissue-specific methylation.
* [Phyloepigenomic comparison of great apes reveals a correlation between somatic and germline methylation states (Martin2011)](http://genome.cshlp.org/content/21/12/2049.long) digestion with a methylation-sensitive enzyme; Neutrophils; human, chimp, orangutan;  The methylomes show a high degree of conservation; methylation states of ~10% of CpG island-like regions differ significantly between human and chimp; The differences are not associated with changes in CG content; The differences recapitulate the known phylogenetic relationship of the three species;  CG decay indicates that somatic methylation states are highly related to germline states.
* [A genome-wide study of DNA methylation patterns and gene expression levels in multiple human and chimpanzee tissues (Pai et al. 2011)](http://dx.plos.org/10.1371/journal.pgen.1001316.g004) suggested that DNA methylation plays a highly conserved tissue-specific role in human and chimpanzees. 
* [Sperm Methylation Profiles Reveal Features of Epigenetic Inheritance and Evolution in Primates (Molaro 2011)](http://www.sciencedirect.com/science/article/pii/S0092867411009421) Question: how patterns of DNA methylation differ between closely related species and whether such differences contribute to species-specific phenotypes. (1) Human vs Chimp: Comparing methylomes of human and chimp sperm revealed a subset of differentially methylated promoters and strikingly divergent methylation in retrotransposon subfamilies, with an evolutionary impact that is apparent in the underlying genomic sequence. (2) male germ cells vs somatic cells: the features that determine DNA methylation patterns differ between male germ cells and somatic cells.

* [Divergent whole-genome methylation maps of human and chimpanzee brains reveal epigenetic basis of human regulatory evolution (Zeng et al. 2012)](http://www.sciencedirect.com/science/article/pii/S0002929712004107): Levels and patterns of DNA methylation vary across individuals within species according to the age and the sex of the individuals. Differentially methylated genes are strikingly enriched with loci associated with neurological disorders, psychological disorders, and cancers.
* [Dynamics of DNA Methylation in Recent Human and Great Ape Evolution (Hernando-Herraez et al. 2013)](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003763) used Illumina Methylation450 BeadChips to profile DNA methylation genome-wide in blood-derived DNA from a total of 9 humans and 23 wild-born individuals of different species and sub-species of chimpanzee, bonobo, gorilla and orangutan, (1) methylation values recapitulate the known phylogenetic relationships of the species (2) a significant positive relationship between the rate of coding variation and alterations of methylation at the promoter level
* [The interplay between DNA methylation and sequence divergence in recent human evolution (Hernando-Herraez et al. 2015)](http://nar.oxfordjournals.org/content/43/17/8204) WGBS data; whole blood; human, chimp, gorilla, orangutan; We identified 8,952,000 CpG positions shared among the four species in autosomal chromosomes, this data set was used for further analysis. (1) inferred TFBS in human-specific DMRs, using the CENTIPEDE algorithm; observed a significant increase in the frequency of human-specific substitutions in predicted TFBS compared with TFBS in the background set (2)the acquisition of DNA hypermethylation in the human lineage is frequently coupled with a rapid evolution at nucleotide level in the neighborhood of these CpG sites
* [Comparative Methylome Analyses Identify Epigenetic Regulatory Loci of Human Brain Evolution (Mendizabal et al. 2016)](https://academic.oup.com/mbe/article/33/11/2947/2272475/Comparative-Methylome-Analyses-Identify-Epigenetic) brains of humans, chimpanzees and also rhesus macaques (added outgroup compared with a previous study); human-specific DMRs validated in multiple individuals from 5 primate species; over half of the newly identified DMRs locate in intergenic regions or gene bodies; DMRs are enriched in active chromatin loops


## Sperm epigenome and development
* [Repressive and active histone methylation mark distinct promoters in human and mouse spermatozoa (Brykczynska 2010)](http://www.nature.com/nsmb/journal/v17/n6/full/nsmb.1821.html) 
  - Trithorax-mediated H3K4me2 and Polycomb-mediated H3K27me3 showed methylation-specific distributions at regulatory regions in human spermatozoa 
  - H3K4me2 marks genes relevant in spermatogenesis and cellular homeostasis
  - H3K27me3 marks developmental regulators in sperm
  - both histone modifications were largely mutually exclusive with DNA methylation 
  
* [Mouse models of male infertility (Cooke 2002)](http://www.nature.com/nrg/journal/v3/n10/full/nrg911.html)
  - Spermatogenesis: development of mature spermatozoa from diploid SPERMATOGONIAL cells — takes ~75 days in man and 35 days in mice 
  - During fetal life
    - 'primordial' germ cell population in the fetal extra-embryonic membranes (at 7 d.p.c. in mouse).
    - Thereafter, they migrate into the gonadal area where, in males, they become associated with differentiating Sertoli cells and form structures known as testis cords (in the mouse, this happens by 12.5 d.p.c.)
  - [Genes implicated in germ-cell meiosis by mouse knockout studies](http://www.nature.com/nrg/journal/v3/n10/fig_tab/nrg911_T3.html)
  - Spermiogenesis: 
    - genes highly/dynamically expressed in spermatid, e.g. Crem, Tlf, Ube2a/b (DNA repair)
    - formation of the ACROSOME
    - somatic histones are replaced sequentially by transition proteins (Tnp1 and Tnp2) and, finally, by highly basic protamines


## Population Epigenetics
* [Population epigenetics. (Richards. 2008)](http://www.ncbi.nlm.nih.gov/pubmed/18337082) A review. 


#### Transgenerational epigenetic inheritance
* [Epigenetic inheritance at the agouti locus in the mouse. (Morgan et al. 1999)](http://www.nature.com.libproxy.usc.edu/ng/journal/v23/n3/full/ng1199_314.html) showed transgenerational epigenetic inheritance resulted from incomplete erasure of an epigenetic modification.
* [Transgenerational inheritance of epigenetic states at the murine Axin(Fu) allele occurs after maternal and paternal transmission. (Rakyan et al. 2003)](http://www.pnas.org/content/100/5/2538.long) found that the methylation state of Axin-Fu in mature sperm reflects the methylation state of the allele in the somatic tissue of the animal, suggesting that it does not undergo epigenetic reprogramming during gametogenesis.
* [Germline epimutation of MLH1 in individuals with multiple cancers. (Suter et al. 2004)](http://www.nature.com.libproxy.usc.edu/ng/journal/v36/n5/abs/ng1342.html) reports two individuals with soma-wide, allele-specific and mosaic hypermethylation of the DNA mismatch repair gene MLH1.  **Epimutation inheritance and cancer**
* [Inheritance of a cancer-associated MLH1 germ-line epimutation. (Hitchins et al. 2007)](http://www.nejm.org/doi/full/10.1056/NEJMoa064522)
  
  > The transmission of epialleles in humans continues to be a topic of debate. The authors provide evidence for transmission of a silenced and hypermethylated epiallele of the tumor suppressor gene MLH1 in a human pedigree.


#### Natural epigenetic variation
* [DNA Methylation Profiling of the Human Major Histocompatibility Complex: A Pilot Study for the Human Epigenome Project. (Rakyan et al. 2004)](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC529316/)
  
  > "Our analysis of DNA methylation levels within the major histocompatibility complex ... reveals a bimodal distribution of methylation profiles, tissue specificity, inter-individual variation, and correlation with independent gene expression data."
* [X-inactivation profile reveals extensive variability in X-linked gene expression in females (Carrel and Willard. 2005)](http://www.nature.com.libproxy.usc.edu/nature/journal/v434/n7031/full/nature03479.html): about 15% of X-linked genes escape inactivation to some degree. An additional 10% of X-linked genes show variable patterns of inactivation and are expressed to different extents from some inactive X chromosomes.
* [Widespread Monoallelic Expression on Human Autosomes. (Gimelbrant et al. 2007)](http://www.sciencemag.org.libproxy.usc.edu/content/318/5853/1136)
  
  > assessed allele-specific transcription of about 4000 human genes in clonal cell lines and found that more than 300 were subject to random monoallelic expression. ... Unexpectedly widespread monoallelic expression suggests a mechanism that generates diversity in individual cells and their clonal descendants. 
* [Comparing the DNA hypermethylome with gene mutations in human colorectal cancer. (Schuebel et al. 2007)](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.0030157):the number of epigenetically inactivated genes in a tumor cell is much larger than the number of genes disrupted by mutations.
* [Heritable rather than age-related environmental and stochastic factors dominate variation in DNA methylation of the human IGF2/H19 locus.(Heijmans et al. 2007)](http://www.ncbi.nlm.nih.gov/pubmed/17339271): ***Genetic source***
  
  > "variation in DNA methylation of the IGF2/H19 locus is mainly determined by heritable factors and single nucleotide polymorphisms (SNPs) in cis, rather than the cumulative effect of environmental and stochastic factors occurring with age."
* [An association between variants in the IGF2 gene and Beckwith-Wiedemann syndrome: interaction between genotype and epigenotype. (Murrell et al. 2004)](http://www.ncbi.nlm.nih.gov/pubmed/14645199) ***Genetic source***
* [Epigenetic differences arise during the lifetime of monozygotic twins (Fraga et al. 2005)](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1174919/) ***Environmental source***
* [Intra- and interindividual epigenetic variation in human germ cells. (Flanagan et al. 2006)](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1474120/): intra and inter-individual methylation variation in CpG islands in 6 gene promoters. 
